docs/clustering/index.rst

```diff
@@ -4,8 +4,8 @@
 Clustering
 ==========
 
-One of the benefits of CrateDB is that you can scale out your cluster by adding
-more nodes. But what's the best way to do that?
+One of the benefits of CrateDB is that you can scale your cluster by adding
+more nodes.
```

Comment:
I would keep "out". "out" here is being used in contrast to "up"

- "out" - scaling horizontally (one of CrateDB's selling points. something that you can do very easily using containers or VMS)
- "up" - scaling vertically (how most RDMBSes scale. and a complete PITA)

see https://stackoverflow.com/a/11715598

if you think we should clarify this in the text in some way, I'd be happy with that. but it's an important enough thing to highlight (in some way) in the intro to this section

thinking aloud...

perhaps this is a good candidate for an entry in the glossary (which we don't have yet). we could, perhaps, create the new glossary page under the appendices section, and add the following terms:

- scaling up (increasing capacity using vertical scaling -- link to "vertical scaling")
- scaling down (decreasing capacity using vertical scaling -- link to "vertical scaling")
- scaling out (increasing capacity using horizontal scaling -- link to "horizontal scaling")
- scaling in (decreasing capacity using horizontal scaling -- link to "horizontal scaling")
- horizontal scaling (full definition)
- vertical scaling (full definition)

that way, you could turn "scale out" in the text her into a link to the appropriate glossary term